src/angular/core/option/optgroup.ts

```diff
@@ -56,6 +56,7 @@ export const SBB_OPTGROUP = new InjectionToken<SbbOptgroup>('SbbOptgroup');
     '[class.sbb-disabled]': 'disabled',
   },
   providers: [{ provide: SBB_OPTGROUP, useExisting: SbbOptgroup }],
+  standalone: false,
```

Comment:
Is there a blocker that prevents us from converting these components to standalone?

Reply:
No. I wasn't aware they are still not standalone and will convert them in a separate step.